### DISCUSSION TITLE
Is there any way to use this library in async?

### DISCUSSION THREAD

**Discussion by TabulateJarl8** [ucfopen/canvasapi#646] [Q&A]
Hey! I was just curious if anyone has found a way to run this library asynchronously. I couldn't find anything in the documentation about doing it within the library itself, so if I missed something or if there's an alternative way to do this I'd really appreciate the help. Thank you!

**Comment by jimmy0017** [comment]
I use celery and then create task to run this asynchronously. 
https://github.com/celery/celery
